src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,29 @@
+/* Experimental text primitive.
+ */
+
+var getMeshMixin = require('../getMeshMixin');
```

Comment:
good question.  was in inherited code, and I believe it's in the A-Frame registerPrimitive sample...